Cargo.toml

```diff
@@ -280,6 +280,12 @@ inherits = "dev"
 debug = 0
 strip = "debuginfo"
 
+# Profile for uv-build
```

Comment:
It'd be nice to talk briefly about the goals here, since we're doing some different things.